azurerm/internal/services/sentinel/resource_arm_sentinel_alert_rule_action.go

```diff
@@ -0,0 +1,168 @@
+package sentinel
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/securityinsight/mgmt/2017-08-01-preview/securityinsight"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	logicParse "github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/logic/parse"
+	logicValidate "github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/logic/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/sentinel/parse"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/sentinel/validate"
+	azSchema "github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/tf/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/timeouts"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmSentinelAlertRuleAction() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmSentinelAlertRuleActionCreate,
+		Read:   resourceArmSentinelAlertRuleActionRead,
+		Delete: resourceArmSentinelAlertRuleActionDelete,
+
+		Importer: azSchema.ValidateResourceIDPriorToImport(func(id string) error {
+			_, err := parse.SentinelAlertRuleActionID(id)
+			return err
+		}),
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(30 * time.Minute),
+			Read:   schema.DefaultTimeout(5 * time.Minute),
+			Delete: schema.DefaultTimeout(30 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringIsNotEmpty,
+			},
+
+			"rule_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.SentinelAlertRuleID,
+			},
+
+			"logic_app_id": {
```

Comment:
it might be worth making this `logic_app_workflow_id`?

Reply:
Yes, that was what I intended to do. But later I noticed in `azurerm_logic_app_trigger_xxx` (e.g. [this](https://www.terraform.io/docs/providers/azurerm/r/logic_app_trigger_custom.html#logic_app_id)) the same field is named after `logic_app_id`, so I follow that convention to be consistent.